build.sbt

```diff
@@ -46,20 +46,13 @@ lazy val defaults = Seq(
   sonatypeProfileName := "com.typesafe",
   scalaVersion := crossScalaVersions.value.last,
   crossScalaVersions := {
-    val earliest211 = 6
-    val latest211 = 12
-    val latest212 = 13
-    val latest213 = 4
-    val skipVersions = Set("2.11.9", "2.11.10")
-    val scala211Versions =
-      (earliest211 to latest211)
-        .map(i => s"2.11.$i")
-        .filterNot(skipVersions.contains(_))
-    ifJavaVersion(_ < 8) {
-      scala211Versions
-    } {
-      scala211Versions ++ (0 to latest212).map(i => s"2.12.$i") ++ (0 to latest213).map(i => s"2.13.$i")
-    }
+    // Remember to keep this list aligned with .travis.yml
+    // 2.11.6 is the first to be supported and we skip 2.11.9 and 2.11.10
+    val supportedScala211Versions = Seq("2.11.6", "2.11.7", "2.11.8", "2.11.11", "2.11.12")
+    // Scala 2.12.[0-2] are not supported
+    val supportedScala212Versions = (3 to 13).map(p => s"2.12.$p")
```

Comment:
Scala `2.12.[0-2]` versions were included before, but the code does not even compile for those versions, which explains they weren't listed in `.travis.yml`.